### PR TITLE
Remove traces of migrations:generate-from-schema

### PR DESCRIPTION
**Pull request by PVince81**
Remove traces of occ migrations:generate-from-schema which was removed
in NC 22.

Also adjust text that specifies that now ALL migrations:* commands are only
available in debug mode.

### CODE REVIEW

**Review by nickvergessen — APPROVED** [review]

File: developer_manual/app_publishing_maintenance/upgrade-guide.rst [hunk 1]
```diff
@@ -79,11 +79,6 @@ The original `PSR-0` standard was deprecated in 2014 and therefore the support f
 .. _`PSR-0`: https://www.php-fig.org/psr/psr-0/
 .. _`PSR-4`: https://www.php-fig.org/psr/psr-4/
 
-Last version with database.xml support and migration
```
Comment (ChristophWurst) [inline developer_manual/app_publishing_maintenance/upgrade-guide.rst:82]: could you please revert this change? This is the upgrade section for people who update their app for Nextcloud 21. This information is still true. Only in 22 this was dropped.

Comment (ChristophWurst) [reply developer_manual/app_publishing_maintenance/upgrade-guide.rst:82]: instead open a new section for 22 and document that this is now gone.

Comment (PVince81) [reply developer_manual/app_publishing_maintenance/upgrade-guide.rst:82]: can do, but I'll have to remove the link because the section about the occ command has been removed.
not sure if we should create a link between different versions of the documentation ?!

Comment (ChristophWurst) [reply developer_manual/app_publishing_maintenance/upgrade-guide.rst:82]: Just drop the link. Developers should be able to figure out the rest :)

Comment (PVince81) [reply developer_manual/app_publishing_maintenance/upgrade-guide.rst:82]: PR here: https://github.com/nextcloud/documentation/pull/6874
